astropy_bot/tests/test_autolabel.py

```diff
@@ -0,0 +1,77 @@
+# For these tests, we patch the repo and pull request handler directly rather
+# than the requests to the server, as we assume the repo and pull request
+# handlers are tested inside baldrick.
+
+from unittest.mock import MagicMock
+
+from astropy_bot.autolabel import autolabel
+
+
+class AutolabelBase:
+
+    all_labels = ['analytic_functions', 'config', 'constants',
+                  'convolution', 'coordinates', 'cosmology',
+                  'io.ascii', 'io.fits', 'io.misc', 'io.misc.asdf',
+                  'io.registry', 'io.votable', 'modeling', 'nddata',
+                  'samp', 'stats', 'table', 'time', 'timeseries',
+                  'uncertainty', 'units', 'utils', 'visualization',
+                  'visualization.wcsaxes', 'vo', 'vo.conesearch',
+                  'wcs', 'wcs.wcsapi']
+
+    def setup_method(self, method):
+
+        self.pr_handler = MagicMock()
+        self.pr_handler.number = 1234
+        self.pr_handler.labels = ['Docs']
```

Comment:
From the tests, it is unclear to me whether this is supposed to _add_ "Docs" label instead of just keeping it there. For example, if I modified only documentation, will it auto-add "Docs" label?

Reply:
I think we don't do anything with the Docs label for now. I'll go ahead and merge this and let's see what happens :)